caas/kubernetes/provider/k8s.go

```diff
@@ -75,6 +75,16 @@ const (
 
 	// OperatorPodIPEnvName is the environment name for operator pod IP.
 	OperatorPodIPEnvName = "JUJU_OPERATOR_POD_IP"
+
+	// OperatorPodIPEnvName is the environment name for operator service IP.
+	OperatorServiceIPEnvName = "JUJU_OPERATOR_SERVICE_IP"
+
+	// OperatorInfoFile is the file containing info about the operator,
+	// copied to the workload pod so the hook tools and juju-run can function.
+	OperatorInfoFile = "operator.yaml"
+
+	// JujuRunServerSocketPort is the port used by juju run callbacks.
+	JujuRunServerSocketPort = 30666
```

Comment:
so we maximumly support 667 units per application before we enhance the action sockets.

Reply:
we will be soon avoiding the use of unit num is the port as we need to suport concurrent actions